app/assets/stylesheets/pages.css.scss

```diff
@@ -13,6 +13,14 @@ $include-html-global-classes: false;
   font-weight: bold;
 }
 
+#sponsor-map-jumbotron {
+  height: 100px;
+
+  @media #{$large-up} {
+    height: 250px;
+  }
+}
+
```

Comment:
Can you give this a more descriptive name than `#sponsor-map-jumbotron`? This map isn't a jumbotron.